src/global/signal_handler.cc

```diff
@@ -152,6 +153,35 @@ void install_standard_sighandlers(void)
 #include "common/Thread.h"
 #include <errno.h>
 
+void get_name_by_pid(pid_t pid, char *task_name)
+{
+  char proc_pid_path[BUF_SIZE] = {0};
+  snprintf(proc_pid_path, BUF_SIZE, "/proc/%d/cmdline", pid);
+  int file_fd = open(proc_pid_path, O_RDONLY);
+
+  if ( file_fd < 0 ) {
+    perror("open");
```

Comment:
do we have the "stderr" opened as a daemon? thought what we have is nothing but `derr`.